include/mqtt/endpoint.hpp

```diff
@@ -6316,21 +6314,17 @@ class endpoint : public std::enable_shared_from_this<endpoint<Socket, Mutex, Loc
     void acquired_async_subscribe(
         packet_id_t packet_id,
         std::string topic_name,
-        std::uint8_t option,
+        std::uint8_t options,
```

Comment:
I use `option` many other places intentionally. The parameter name `options` at https://github.com/redboltz/mqtt_cpp/blob/master/include/mqtt/endpoint.hpp#L6278 is typo.

## Current policy

MQTT spec defines `options`. It contains multiple information but one byte. The parameter `option` is MQTT's `options`. I want to reserve postfix `s` (plural) for collection such as vector. It is similar to `topic` and `topics`.

## Possible choice
I know it is confusing. We can use some other postfix for collection instead of `s`. For example, `_col` (means collection). `topic` and `topic_col`, `options` and `options_col`.
I want to keep consistency. Perhaps my code might already have consistency violation, but my policy is distinguishing single object or collection of objects.

Reply:
Ok. Will fix.

Reply:
Fixed